DatadogSessionReplay/Sources/Processor/Processor.swift

```diff
@@ -52,9 +52,18 @@ internal class Processor: Processing {
     var interceptWireframes: (([SRWireframe]) -> Void)? = nil
     #endif
 
-    init(queue: Queue, writer: Writing) {
+    private var srContextPublisher: SRContextPublisher
+
+    private var recordsCount: [String: Int64] = [:]
```

Comment:
suggestion/ Similar ambiguity here - it doesn't say what the `String` is.
```suggestion
    private var recordsCountByViewID: [String: Int64] = [:]
```
or
```suggestion
    /// Records count by view id.
    private var recordsCount: [String: Int64] = [:]
```